src/api/local-storage.js

```diff
@@ -0,0 +1,64 @@
+class localStorageApi {
+
+  static getUsers() {
+    var existingUsers = JSON.parse(localStorage.getItem("allStoredUsers")) || [];
+    console.log("getUsers() current existing stored users = ", existingUsers);
+    return existingUsers;
+ }
+ 
+  static createUser(newUser) {
+    var existingUsers = localStorageApi.getUsers();
```

Comment:
We've basically moved away from using `var` entirely. Stick with `let` and `const`.
```suggestion
    const existingUsers = localStorageApi.getUsers();
```